rholang/src/test/scala/coop/rchain/rholang/interpreter/matcher/MatchTest.scala

```diff
@@ -447,18 +446,15 @@ class VarMatcherSpec extends FlatSpec with Matchers with TimeLimits with TripleE
     assertSpatialMatch(targetPar, patternPar, expectedResult)
 
     val nonMatchingPattern: Expr =
-      ParSet(
-        Seq(GInt(2), GInt(5), EVar(FreeVar(0)), EVar(Wildcard(WildcardMsg()))),
-        connectiveUsed = true
-      )
+      ParSet(Seq(GInt(2), GInt(5), EVar(FreeVar(0)), EVar(Wildcard(WildcardMsg()))))
     assertSpatialMatch(target, nonMatchingPattern, None)
   }
 
   it should "work with wildcard remainders" in {
     val targetElements = Seq[Par](GInt(1), GInt(2), GInt(3), GInt(4), GInt(5))
     val target: Expr   = ParSet(targetElements)
     val pattern: Expr =
-      ParSet(Seq(GInt(1), GInt(4)), connectiveUsed = true, remainder = Wildcard(WildcardMsg()))
+      ParSet(Seq(GInt(1), GInt(4)), remainder = Wildcard(WildcardMsg()))
```

Comment:
~I think this will no longer have `connectiveUsed` in your implementeation.~

It will. But we should add a test if no test of ParSet fails when you don't include remainder.

We should add a generative test of the form:
```scala
forAll { (a: Expr, b: Expr, r: Var) =>
   ParSet(Seq(a, b), remainer = r).connectiveUsed == Seq(a, b, r).conains(_.connectiveUsed) //assumin Var has connectiveUsed == true...
}
```
or sth like that

Reply:
There is a test for that: https://github.com/rchain/rchain/blob/72c6dff860ee6147443feac60b5fddf8367d058f/rholang/src/test/scala/coop/rchain/rholang/interpreter/matcher/MatchTest.scala#L434